inst/templates/code-cpp11.cpp

```diff
@@ -0,0 +1,5 @@
+#include <cpp11.h>
+using namespace cpp11;
+
+[[cpp11::export]]
+void fun() {}
```

Comment:
It seemed useful to me to have an example of the exporting syntax in the template file, let me know if you don't agree, currently the Rcpp example does not have this.